libs/grok/src/test/java/org/elasticsearch/grok/GrokTests.java

```diff
@@ -439,4 +439,13 @@ public void testExponentialExpressions() {
         run.set(false);
         assertThat(e.getMessage(), equalTo("grok pattern matching was interrupted after [200] ms"));
     }
+
+    public void testUnicodeFieldnames() {
+        for(String fieldName : Arrays.asList("@metadata", "@metädata", "@metädat[a]")) {
+            String line = "foo";
```

Comment:
Why is this put into a local var? It's only used once.

Reply:
fixed